frontend/src/lib/components/Cards/InsightCard/TileOverridesWarning.tsx

```diff
@@ -0,0 +1,19 @@
+import { IconWarning } from '@posthog/icons'
+
+import { Tooltip } from 'lib/lemon-ui/Tooltip'
+
+export function TileOverridesWarning(): JSX.Element | null {
+    return (
+        <Tooltip
+            title={
+                <div className="flex items-center gap-1">
+                    <span>Tile filters override all other filters</span>
```

Comment:
In the future it might be cool to render the filters in the tooltip so they can see what they are without needing to open the modal.